tests/converter_tests/test_md2html_converter.py

```diff
@@ -166,6 +167,68 @@ def test_tw(self):
             self.assertTrue(os.path.isfile(file_path), 'file not found: {0}'
                             .format(file_to_verify))
 
+    def test_tn(self):
+        """
+        Runs the converter and verifies the output
+        """
+
+        # given
+        file_name = 'en_tn.zip'
+
+        # when
+        self.doTransformTn(file_name)
+
+        # then
+        self.assertTrue(os.path.isfile(self.out_zip_file), "There was no output zip file produced.")
+        self.assertIsNotNone(self.return_val, "There was no return value.")
+        self.out_dir = tempfile.mkdtemp(prefix='tw_')
+        unzip(self.out_zip_file, self.out_dir)
+        remove(self.out_zip_file)
+
+        files_to_verify = ['00-toc.html', 'manifest.yaml', 'index.json']
+        for dir in BOOK_NUMBERS:
+            book = '{0}-{1}'.format(BOOK_NUMBERS[dir], dir.upper())
+            file = '{0}.html'.format(book)
```

Comment:
The reason `file` comes up even as orange here is its a reserved word in Python. Change to filename.